src/main/java/uk/gov/di/ipv/service/CredentialIssuerService.java

```diff
@@ -42,14 +58,16 @@ public BearerAccessToken exchangeCodeForToken(CredentialIssuerRequestDto request
                         errorResponse.getErrorObject(),
                         new ErrorObject("unknown", "unknown")
                 );
-                throw new CredentialIssuerException(String.format("%s: %s", errorObject.getCode(), errorObject.getDescription()));
+                LOGGER.error(String.format("%s: %s", errorObject.getCode(), errorObject.getDescription()));
```

Comment:
You could also use `LOGGER.error("{}: {}", errorObject.getCode(), errorObject.getDescription());` here